src/modules/grep/grepApi.tsx

```diff
@@ -13,12 +13,32 @@ import {
 } from '../../util/apiHelpers';
 import handleError from '../../util/handleError';
 
+interface Tekst {
+  tekst: {
+    spraak: string;
+    verdi: string;
+  }[];
+}
+
+interface Title {
+  spraak: string;
+  verdi: string;
+}
```

Comment:
Er ikkje dette berre en array av samme typen?

```suggestion
interface Tekster {
  tekst: Tekst[];
}

interface Tekst {
  spraak: string;
  verdi: string;
}
```